src/schema/v2/object_identification.ts

```diff
@@ -245,6 +245,14 @@ export const InternalIDFields: GraphQLFieldConfigMap<any, ResolverContext> = {
   },
 }
 
+export const InternalIDField: GraphQLFieldConfigMap<any, ResolverContext> = {
```

Comment:
**suggestion / non-blocking**: Can we also reuse this definition within the `GraphQLFieldConfigMap`s defined above?

For example:

```ts
export const InternalIDFields: GraphQLFieldConfigMap<any, ResolverContext> = {
  id: GlobalIDField,
  internalID: InternalIDField
}
```